### DISCUSSION TITLE
New 0.x release?

### DISCUSSION THREAD

**Discussion by vfazio** [encode/httpx#2441] [General] [answered]
I realize 1.x is on the horizon, but it sounds like there may be some blockers: 

https://github.com/encode/httpx/milestone/1
https://github.com/encode/httpx/discussions/2279

Is there any plan for a new point release advertising 3.11 support in the interim?

**Comment by lovelydinosaur** [comment, marked as answer]
Done... https://pypi.org/project/httpx/

See PR https://github.com/encode/httpx/pull/2442

**Reply by vfazio** [reply]
Thanks!
